packages/flutter/lib/src/widgets/async.dart

```diff
@@ -391,6 +407,16 @@ class StreamBuilder<T> extends StreamBuilderBase<T, AsyncSnapshot<T>> {
 /// * `new AsyncSnapshot<String>.withData(ConnectionState.waiting, null)`
 /// * `new AsyncSnapshot<String>.withError(ConnectionState.done, 'some error')`
 ///
+/// By default, the initial snapshot delivered to the [builder] will not contain
```

Comment:
As above. Maybe say
```dart
/// The initial snapshot data can be controlled by specifying [initialData]. You would use
/// this facility to ensure that if the [builder] is invoked before the future completes, the
/// snapshot carries data of your choice rather than the default null value.
```